packages/react-mutation-mapper/src/store/DefaultMutationMapperStore.ts

```diff
@@ -620,6 +623,35 @@ class DefaultMutationMapperStore implements MutationMapperStore
         }
     }
 
+    readonly civicGenes: MobxPromise<ICivicGene | undefined> = remoteData({
+        await: () => [
+            this.mutationData,
+        ],
+        invoke: async() => this.config.enableCivic ?
+            fetchCivicGenes(this.mutationData.result || [], this.getDefaultEntrezGeneId) : {},
+        onError: () => {
+            // fail silently
+        }
+    }, undefined);
```

Comment:
i think the default is undefined by default, so, not sure this makes sense.  the intention of the default was to provide an empty array or object so that consumer code didn't have to do check against undefined